### PR TITLE
Esm

### LINKED ISSUE #17: ES module version
It would be nice if this module was available as a standard module. 

I'm happy to provide a PR

### PR DESCRIPTION
**Pull request by bennypowers**
the first commit is the real deal, the last two are optional and can be dropped if you like.

If you like, I'm happy to es6-up the source lib in a separate PR as well

fixes #17

### CODE REVIEW

File: idb-keyval.js [hunk 1]
```diff
@@ -1,91 +1,80 @@
...
+    }).then(function() {
+      return keys;
     });
-  } else {
-    self.idbKeyval = idbKeyval;
   }
-}());
+};
+
+export default idbKeyval
```
Comment (jakearchibald) [inline idb-keyval.js]: Semicolon 😄

File: package.json [hunk 2]
```diff
@@ -2,9 +2,11 @@
   "name": "idb-keyval",
   "version": "2.3.0",
   "description": "A super-simple-small keyval store built on top of IndexedDB",
-  "main": "idb-keyval.js",
+  "main": "dist/idb-keyval-cjs.js",
+  "module": "idb-keyval.js",
   "typings": "typings.d.ts",
   "scripts": {
+    "postinstall": "npx rollup -c",
     "build": "uglifyjs idb-keyval.js --screw-ie8 -mc --output dist/idb-keyval-min.js"
```
Comment (jakearchibald) [inline package.json]: I guess this is broken now?

File: dist/idb-keyval.mjs [hunk 3]
```diff
@@ -0,0 +1,80 @@
+var db;
```
Comment (jakearchibald) [inline dist/idb-keyval.mjs]: Do we need this file? Isn't it the same as the root?

File: package.json (hunk 2 above)
Comment (bennypowers) [reply package.json]: yeah. we've got https://www.npmjs.com/package/rollup-plugin-uglify I will add that in the next commit

File: dist/idb-keyval.mjs (hunk 3 above)
Comment (bennypowers) [reply dist/idb-keyval.mjs]: My idea here was to help node users that need the lib

OTOH, I don't think idb is a think in node world, so yeah we don't need it.

**Comment by jakearchibald** [comment]
Merged with a couple of tweaks, and published to npm. Thank you!

**Pull request merged**